unison-runtime/src/Unison/Runtime/Stack.hs

```diff
@@ -581,7 +581,7 @@ pattern IntVal i <- (matchIntVal -> Just i)
 
 matchBoolVal :: Val -> Maybe Bool
 matchBoolVal = \case
-  (BoxedVal (Enum r t)) | r == Ty.booleanRef -> Just (t == TT.falseTag)
+  (BoxedVal (Enum r t)) | r == Ty.booleanRef -> Just (t == TT.trueTag)
```

Comment:
If this was flipped, how come it wasn't noticed before? Was this code just unused until now?

Reply:
Yeah, probably not used in this mode.